hs_core/templates/pages/genericresource.html

```diff
@@ -2529,7 +2530,9 @@ <h3 style="color: red" id="file-type-error"></h3>
                         }
                         setPointerEvents(true);
                     },
-                    error: function (XMLHttpRequest, textStatus, errorThrown) {
+                    error: function(XMLHttpRequest, textStatus, errorThrown){
+                        $("#div-invite-people").find(".label-danger").remove(); // Remove previous alerts
```

Comment:
What happens if .label-danger class element is not appended yet and thus the find() returns null? Will it raise an error when you call remove() on a null object? Probably it is better to do a check here so as to only call remove() when the find result is not null.